pkg/trace/api/profiles.go

```diff
@@ -68,7 +74,20 @@ func (r *HTTPReceiver) profileProxyHandler() http.Handler {
 	if err != nil {
 		return errorHandler(err)
 	}
-	tags := fmt.Sprintf("host:%s,default_env:%s", r.conf.Hostname, r.conf.DefaultEnv)
+	ver, err := version.Agent()
+	verStr := "unknown"
+	if err == nil {
+		verStr = ver.GetNumberAndPre()
+	}
+
+	tags := fmt.Sprintf("host:%s,default_env:%s,agent_version:%s", r.conf.Hostname, r.conf.DefaultEnv, verStr)
+
+	// Note: check for Fargate explicitly, need identifying tags for host tracking
+	if fargate.IsFargateInstance(context.Background()) {
+		orchestratorTag := fmt.Sprintf("orchestrator:fargate_%s",
```

Comment:
```suggestion
		tag := fmt.Sprintf("orchestrator:fargate_%s",
```

Reply:
It is a good idea to add this kind of tag to get visibility about the container environment, but I think if we add this kind of tag it should be managed by the agent tagger to share the same tag across all the workload (metrics, logs...)

Reply:
Thanks, we would like this tag to be added more generally, but there a bit of context around this change. Mainly, we are trying to land this change before the code freeze in the next day to help fix a billing issue. There are a lot more changes for this to be standardized + used in the tagger, so we want to try this in profiling and understand the fargate situation 100% before standardizing more generally. Is that ok?